lib/types/command_payload.ex

```diff
@@ -79,6 +79,7 @@ defmodule Kadena.Types.CommandPayload do
 
   @spec validate_meta(meta :: meta()) :: validation()
   defp validate_meta(%MetaData{} = meta), do: {:ok, meta}
+  defp validate_meta(nil), do: {:ok, %MetaData{}}
```

Comment:
@Odraxs @EdwinGuayacan a better approach would be to initialize the empty metadata structure like this `Metadata.new()`.